src/Components/sidebar.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import styles from './sidebar.module.css'
+import { SidebarData } from './SidebarData'
+type Props = {}
+
+export default function Sidebar({ }: Props) {
+  return (
+    <ul className={styles.navbar}>
+      {SidebarData.map((item, index) => {
+        return (
+          <li key={index} className={styles.listItem}>
+            <Link to={item.path} draggable={false}>
+              <div className={styles.sidebarContent}>
+                <div className={styles.icon}>{item.icon}</div>
```

Comment:
consider removing the wrapping div and target the svg directly using css direct child selector